### PR TITLE
Move staff pets show image

### PR DESCRIPTION
**Pull request by Aaryanpal**
# 🔗 Issue
#862 

# ✍️ Description
- [x] Remove the image to the right side
- [x] Add thumbnail image to the header space of the page layout per figma design
- [x] Page content extends full page width in lieu of removed image

# 📷 Screenshots/Demos

https://github.com/user-attachments/assets/d2ab1990-5170-4a0f-98cf-ed8b6abcc1bd

### CODE REVIEW

File: app/views/organizations/staff/pets/show.html.erb [hunk 1]
```diff
@@ -1,5 +1,13 @@
 <%= render DashboardPageComponent.new(crumb: :dashboard_pet, crumb_options: [@pet]) do |c| %>
-  <% c.with_header_title { @pet.name } %>
+  <% c.with_header_title do %>
```
Comment (kasugaijin) [inline app/views/organizations/staff/pets/show.html.erb:2]: @Aaryanpal we just made a new pet avatar component, actually. Could you please use this here instead?

app/components/pet_avatar_component.rb

Comment (Aaryanpal) [reply app/views/organizations/staff/pets/show.html.erb:2]: okay

Comment (Aaryanpal) [reply app/views/organizations/staff/pets/show.html.erb:2]: [Screencast from 2024-08-28 20-06-53.webm](https://github.com/user-attachments/assets/672b1140-7df1-4606-ba5a-68363a7be735)

@kasugaijin, is it okay to show the initial letter of the pet's name when an image is not available? Also, should we update the pets index and fix the images to ensure they appear rounded?

image

**Review by kasugaijin — APPROVED** [review]
Looks good @Aaryanpal 
Thanks for addigin more options to the component. 
I think showing the pet initial is fine for now. We can update this later if we need to show something else. 

I do agree that we should use this avatar in the Pet index table - so if you want to work on that, please create a new issue and have a go!

Thanks

**Pull request merged**